test/e2e/generate_kube_test.go

```diff
@@ -54,7 +54,7 @@ var _ = Describe("Podman generate kube", func() {
 	})
 
 	It("podman generate kube on container", func() {
-		session := podmanTest.RunTopContainer("top")
+		session := podmanTest.Podman([]string{"run", "--name", "top", "-d", "--http-proxy=false", ALPINE, "top"})
```

Comment:
Did you consider just adding `--http-proxy=false` here instead?  https://github.com/containers/podman/blob/c00d8a27d90da9d83b41458ebedd06e1cc61a105/test/e2e/common_test.go#L441-L442

Reply:
I added `--http-proxy=false` to RunTopContainerWithArgs function and 
checked the tests are all ok with a proxy.
I force push the commit.